app/src/pages/report/index.tsx

```diff
@@ -4,7 +4,6 @@ import PageHeader from '@/components/PageHeader';
 import Container from 'react-bootstrap/Container';
 import Loading from '@/components/Loading';
 import Table from '../../components/report/Table';
-
```

Comment:
Is this intentional? It's nice to have extra whitespace between imports and application logic just for readability :)